dusty/cli/__init__.py

```diff
@@ -124,12 +124,12 @@ def main():
         return run_daemon()
 
     args = docopt(__doc__, options_first=True)
+    if args['-v']:
+        print 'Dusty version {}'.format(constants.VERSION)
+        sys.exit(0)
```

Comment:
this fixes it because of docopt?   like if a did dusty logs -v it would not show up in args -v?